bin/systemd_module_migration.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# we rquire hub later on for the PR
+if ! which hub > /dev/null ; then
+  exit 1
+fi
+
+# identify all modules with systemd in .fixtures.yml
+while read -r file; do
+  sed -i 's#camptocamp/puppet-systemd#voxpupuli/puppet-systemd#' "${file}"
+done < <(grep -l "camptocamp/puppet-systemd" modules/voxpupuli/puppet-*/.fixtures.yml)
+
+# identify all modules with systemd from camptocamp in their metadata.json
+while read -r file; do
+  sed -i 's#camptocamp[-,/]systemd#puppet/systemd#' "${file}"
+done < <(grep -l "camptocamp[-,/]systemd" modules/voxpupuli/puppet-*/metadata.json)
+
+# bump the metadata because there was a recent major release
+./bin/bump-dependency-upper-bound puppet/systemd 4.0.0 modules/voxpupuli/puppet-*/metadata.json
+
+# commit and create a PR
+for module in modules/voxpupuli/puppet-* ; do
+  (cd "$module"
+  changed_file=$(git diff --name-only)
+  # last compare doesn't work
+  #if [ "${changed_file}" = ".fixtures.yml" ] || [ "${changed_file}" = "metadata.json" ] || [ "${changed_file}" = ".fixtures.yml metadata.json" ]; then
+  if [ "${changed_file}" != "" ]; then
+    git checkout -b systemd_voxpupuli origin/master
+    git commit -am 'switch from camptocamp/systemd to voxpupuli/systemd'
+    git push --set-upstream origin systemd_voxpupuli
```

Comment:
I tend to prefer avoid repeating the branch name
```suggestion
    git push --set-upstream origin HEAD
```